Spec/RealtimeClientPresence.swift

```diff
@@ -957,6 +957,94 @@ class RealtimeClientPresence: QuickSpec {
                     expect(compareForNewnessMethodCalls) == 1
                 }
 
+                // RTP2b
+                context("compare for newness") {
+
+                    // RTP2b1
+                    it("presence message has a connectionId which is not an initial substring of its id") {
+                        let options = AblyTests.commonAppSetup()
+
+                        let clientSubscribed = ARTRealtime(options: options)
+                        defer { clientSubscribed.dispose(); clientSubscribed.close() }
+                        let channelSubscribed = clientSubscribed.channels.get("foo")
+                        channelSubscribed.attach()
+
+                        let clientPresentMember = ARTRealtime(options: options)
+                        defer { clientPresentMember.dispose(); clientPresentMember.close() }
+                        let channelPresentMember = clientPresentMember.channels.get("foo")
+
+                        var hasInconsistentConnectionIdMethodCalls = 0
+                        let hook = channelSubscribed.presenceMap.testSuite_injectIntoMethodAfter(NSSelectorFromString("hasInconsistentConnectionId")) {
```

Comment:
@mattheworiordan Yes you're right. I should test behaviour and not implementation details.  I've looked at how Ruby spec is implemented and I changed it: bac7e91.